src/index.js

```diff
@@ -99,7 +99,7 @@ const ApolloClientPlugin = createPlugin({
 
         return forward(operation);
       });
-      const links = getApolloLinks([authMiddleware, connectionLink]);
+      const links = getApolloLinks({token})([authMiddleware, connectionLink]);
```

Comment:
I'm wondering if we can instead pass in context to this function as well. Yes there would be some duplicated code, but I feel that the implementation would be a bit more clean.

Reply:
@KevinGrandon Any example ? I'm not quite clear about context (ApolloContext or KoaContext or HttpContext) ?

Reply:
Possibly something like: 
```
       const links = getApolloLinks([authMiddleware, connectionLink], ctx); 
```

Reply:
Actually, we need to pass all `{endpoint, authKey,...}` to outside as well.

Reply:
Yeah, i got your idea. But i prefer currying in this case, just more clean ?